types/passport/lib/errors/authenticationerror.d.ts

```diff
@@ -0,0 +1,5 @@
+export default class AuthenticationError extends Error {
```

Comment:
This is the wrong kind of export for this [source.](https://github.com/jaredhanson/passport/blob/master/lib/errors/authenticationerror.js#L20)
It should be
```ts
class AuthenticationError extends Error {
    name: 'AuthenticationError';
    message: string;
    status: number;
}
export = AuthenticationError;
```
instead.

Reply:
> This is the wrong kind of export for this [source.](https://github.com/jaredhanson/passport/blob/master/lib/errors/authenticationerror.js#L20)
> It should be
> 
> ```ts
> class AuthenticationError extends Error {
>     name: 'AuthenticationError';
>     message: string;
>     status: number;
> }
> export = AuthenticationError;
> ```
> 
> instead.

Tried it and the linter give me this:
```
ERROR: 2:38  expect  TypeScript@next compile error:
This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.
```

Reply:
Yep, as the error suggests, just set `esModuleInterop` in the test `tsconfig`.